Unified Cloud Navigation screenshot removal and style edits 2 (#191)
hashicorp/terraform-docs-common: website/docs/cloud-docs/run/remote-operations.mdx

> If a speculative plan fails due to an external factor, you can run it again using the "Retry Run" button on its page:

Reviewer: ```suggestion
If a speculative plan fails due to an external factor, click **Retry Run** to try it again.
```
